offchain/schema.graphql

```diff
@@ -1,205 +0,0 @@
-"Data that can be used as proof to validate notices and execute vouchers on the base layer blockchain"
```

Comment:
This file was committed by accident previously. Now, we are generating it on demand in the CI.